### PR TITLE
Youtube page + template initial stub

### PR DESCRIPTION
**Pull request by Pomax**
See: #3684 

This stubs the youtube page + template, with a placeholder youtube_regret_block for content testing

@mmmavis  let's iterate on this?

Regret page:
- https://foundation-mofostaging-pr-3699.herokuapp.com/cms/pages/51/edit/
- https://foundation-mofostaging-pr-3699.herokuapp.com/en/regrat/

### CODE REVIEW

**Comment by mmmavis** [comment]
@Pomax we have to use `nav-newsletter.js` for the nav to work on both mobile and desktop. See example: https://github.com/mozilla/foundation.mozilla.org/blob/master/source/js/main.js#L190

**Comment by Pomax** [comment]
that's fine, we have full control over what actually gets loaded in terms of JS. I just figured there was very little point in loading the entire `main.js` up front =)

File: network-api/networkapi/wagtailpages/models.py [hunk 1]
```diff
@@ -1352,18 +1352,19 @@ class Homepage(FoundationMetadataPageMixin, Page):
...
         'ParticipatePage2',
-        'MiniSiteNameSpace',
+        'PeoplePage',
+        'PrimaryPage',
         'RedirectingPage',
-        'OpportunityPage',
-        'BanneredCampaignPage',
-        'IndexPage',
+        'Styleguide',
+        'YoutubeRegretsPage',
```
Comment (mmmavis) [inline network-api/networkapi/wagtailpages/models.py]: URL-wise the YouTube Regrets page is going to be on `/campaigns/blah`. Let's make `YoutubeRegretsPage` only available as a child page of the campaigns mini site.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_page.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{% extends "pages/base.html" %}
```
Comment (mmmavis) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_page.html]: Can we change the indentation from 4 spaces to 2? It seems to be the convention for `.html` files in Foundation site codebase.

**Pull request merged**